t/plugin/limit-count.t

```diff
@@ -1175,3 +1175,95 @@ passed
 [200, 200, 503, 503]
 --- no_error_log
 [error]
+
+
+
+=== TEST 37: the limit-count plugin is bound to the service and upstream is the domain name
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/services/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "plugins": {
+                        "limit-count": {
+                            "count": 3,
+                            "time_window": 60,
+                            "rejected_code": 503
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "test.com:1980": 1
+                        },
+                        "type": "roundrobin"
+                    }
+                 }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
+
+
+=== TEST 38: add route and bind service
```

Comment:
I think we should merge TEST 38 into TEST 37, so that it is clear that they are both for TEST 39.

Reply:
updated.